app/src/main/java/io/pslab/activity/MultimeterActivity.java

```diff
@@ -133,7 +134,7 @@ public void onClick(View v) {
         Typeface tf = Typeface.createFromAsset(getAssets(), "fonts/digital-7 (italic).ttf");
         quantity.setTypeface(tf);
 
-        String text_quantity = multimeter_data.getString("TextBox", null);
+        String text_quantity = multimeter_data.getString("TextBox", defaultValue);
```

Comment:
May be this can be stringformatted with English Locale to fix the comma dot issue rather than directly using the value.